src/base/File.cc

```diff
@@ -331,7 +332,7 @@ File::lock(const FileOpeningConfig &cfg)
                    " more time(s) after a failure: " << ex.what());
         }
         Must(attemptsLeft); // the catch statement handles the last attempt
-        xusleep(cfg.RetryGapUsec);
+        std::this_thread::sleep_for(std::chrono::microseconds(cfg.RetryGapUsec));
```

Comment:
Would it make sense to have an inline xusleep making this call less verbose?

Reply:
I would like to eventually have the object members store the std::chrono duration types. But that is a lot more work and the scope here is just removing the xusleep hacks.